app/backend/src/couchers/migrations/versions/46e6b6a3d9ed_added_is_cancelled_and_is_deleted_for_.py

```diff
@@ -0,0 +1,26 @@
+"""Added is_cancelled and is_deleted for Events
+
+Revision ID: 46e6b6a3d9ed
+Revises: 88b6bb559332
+Create Date: 2024-05-16 00:44:01.904266
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "46e6b6a3d9ed"
+down_revision = "b5355a1f60f6"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column("event_occurrences", sa.Column("is_cancelled", sa.Boolean(), nullable=False))
+    op.add_column("event_occurrences", sa.Column("is_deleted", sa.Boolean(), nullable=False))
```

Comment:
Maybe add `server_default=sa.text("false")`

Reply:
Good point, will add that later 👍